src/App.js

```diff
@@ -4,12 +4,15 @@ import createAuthContext from './lib/createAuthContext'
 const clientId = process.env.REACT_APP_CLIENT_ID || "8cb4904ae5581ecc2b3a1774"
 const clientSecret = process.env.REACT_APP_CLIENT_SECRET || "b683283462070edbac15a8fdab751ada0f501ab48a5f06aa20aee3be24eac9cc"
 const provider = process.env.REACT_APP_PROVIDER || "https://authenticate.u5auth.com"
+const redirectUri = process.env.REACT_RETURN_URI || "https://localhost"
+const useUrlEncodedForm = process.env.REACT_USE_URLENCODEDFORM || true
 
 const {AuthContext, Authenticated, useToken} = createAuthContext({
   clientId,
-  clientSecret,
   provider,
-  // tokenEndpoint: 'http://localhost:3020/token' // If token endpoint is not "provider + '/token'"
```

Comment:
This should not be removed either, as it is documentation for those that need to customise the token endpoint.

Reply:
I was using the internal files I had used to create my own solution, but it got mixed up with the original. I brought everything back to its original state.